### PR TITLE
Added endpoint for getting comments by article id and tested.

### PR DESCRIPTION
**Pull request by TestMango-sudo**
Added endpoint for getting comments by article id and tested.

### CODE REVIEW

File: controllers/controllers.js [hunk 1]
```diff
@@ -30,3 +29,15 @@ exports.getArticleById = (req, res) => {
     })
 }
 
+exports.getCommentsByArticleId = (req, res) => {
+    const { article_id } = req.params
+    fetchCommentsByArticleId(article_id).then((data) => {
+        if (data.length === 0){
```
Comment (nc-ali-abdallah) [inline controllers/controllers.js:35]: as per the test I said we should include this isn't correct, we want to return the empty array, also if we didn't want to include this we should include it in the model not the controller.

**Review by nc-ali-abdallah — COMMENTED** [review]
I'd recommend looking into toMatchObject or objectContaining jest matchers when it comes to testing they allow for cleaner/readable/more flexible testing.

Make sure you are doing a length check in your tests when you are receiving an array back otherwise you might end up with false passing tests. 

We should also include a 404 test for an existent article ID.
Also a 200 test for a valid article but has no comments (We should return an empty array).

Make sure to remove console logs once you are finished with them.
